api/turing/service/mlp_service.go

```diff
@@ -229,7 +233,7 @@ func (service mlpService) GetEnvironments() ([]merlin.Environment, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), mlpQueryTimeoutSeconds*time.Second)
 	defer cancel()
 
-	environments, resp, err := service.merlinClient.api.EnvironmentApi.EnvironmentsGet(ctx, nil)
+	environments, resp, err := service.merlinClient.api.EnvironmentAPI.EnvironmentsGet(ctx).Execute()
```

Comment:
`EnvironmentsGet` no longer returns the outputs of the API call directly - see https://github.com/caraml-dev/merlin/blob/main/api/client/api_environment.go#L47. We now need to call `Execute()` on the returned object - see https://github.com/caraml-dev/merlin/blob/main/api/client/api_environment.go#L35.